backend/rates/rates.go

```diff
@@ -275,36 +276,42 @@ func (updater *RateUpdater) updateLast(ctx context.Context) {
 	}
 
 	var geckoRates map[string]map[string]float64
-	callErr := updater.geckoLimiter.Call(ctx, "updateLast", func() error {
-		ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
-		defer cancel()
-		res, err := updater.httpClient.Do(req.WithContext(ctx))
-		if err != nil {
-			return errp.WithStack(err)
-		}
-		defer res.Body.Close() //nolint:errcheck
-		if res.StatusCode != http.StatusOK {
-			return errp.Newf("bad response code %d", res.StatusCode)
-		}
-		const max = 10240
-		responseBody, err := io.ReadAll(io.LimitReader(res.Body, max+1))
-		if err != nil {
-			return errp.WithStack(err)
-		}
-		if len(responseBody) > max {
-			return errp.Newf("rates response too long (> %d bytes)", max)
-		}
-		if err := json.Unmarshal(responseBody, &geckoRates); err != nil {
-			return errp.WithMessage(err,
-				fmt.Sprintf("could not parse rates response: %s", string(responseBody)))
-		}
-		return nil
-	})
-	if callErr != nil {
-		updater.log.WithError(callErr).Errorf("updateLast")
+	if err := updater.geckoLimiter.Wait(ctx); err != nil {
+		updater.log.WithError(err).Error("could not wait for rate limiter")
```

Comment:
please prefix all msgs with `updateLast:` like it used to be before, so the error is more easily identified in the logs.